teku/src/main/java/tech/pegasys/teku/cli/subcommand/VoluntaryExitCommand.java

```diff
@@ -227,7 +227,11 @@ private void initialise() {
             new RejectingSlashingProtector(),
             new PublicKeyLoader(),
             asyncRunner,
-            metricsSystem);
+            metricsSystem,
+            config.validatorRestApiConfig().isRestApiEnabled()
+                ? Optional.of(Path.of("/"))
+                : // FIXME: where can we get the path?
```

Comment:
this is why the path should come from that DataDir object i've mentioned. slashing exporter uses the same object, hopefully it all ultimately comes from internals not from merged strings
also can't merge with fixme

Reply:
i think the more likely scenario here is that they don't load mutable validators at all in this way potentially. the other point i would make here is the restapi is guaranteed not to be enabled at this point (we're not running any services)

Reply:
the other option i guess is we always check that folder for voluntary exits case, might be a @ajsutton question - if we have the key and someone's using CLI seems weird if they're mostly managing the keys through a gui...